Translations/pt_BR.po

```diff
@@ -734,59 +732,66 @@ msgstr "Não"
 
 #: Source/DiabloUI/support_lines.cpp:8
 msgid "GOG.com maintains a web site at https://www.gog.com/forum/diablo"
-msgstr ""
+msgstr "GOG.com mantem um site em https://www.gog.com/forum/diablo"
```

Comment:
```suggestion
msgstr "GOG.com mantém um site em https://www.gog.com/forum/diablo"
```

Reply:
I always confuse this one... for some reason I always think the accent was dropped in the last language update, but it's not the case. Thanks for the fix.

Reply:
@julealgon yeah, I still struggle with the "recent" language update too.